replication-api/src/main/java/org/codice/ditto/replication/api/persistence/ReplicationItemManager.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ditto.replication.api.persistence;
+
+import java.util.List;
+import java.util.Optional;
+import org.codice.ditto.replication.api.ReplicationItem;
+
+public interface ReplicationItemManager {
+
+  /**
+   * If present, returns a {@link ReplicationItem} for the given {@link
+   * org.codice.ditto.replication.api.data.Metadata} identified by its.
```

Comment:
❓ is there a reason so many class links are fully qualified in this interfaces java docs?

Reply:
because they are not imported, and if they were imported the formatter would fail saying they are unused.